[ENG-1742] Add pentesting docs (#16)
Oneleet/oneleet-docs: pages/penetration-testing/offerings.mdx

# Penetration Test Offerings

> Multiple types of Penetration Tests/Engagements fall within the expertise of Oneleet’s penetration testing team, some of them being:

Reviewer: ```suggestion
Several types of Penetration Tests/Engagements fall within the expertise of Oneleet’s penetration testing team, including:
```

> - Network Pentesting;
> - Mobile App Pentesting;
> - Web App Pentesting;
> - Wireless Network Pentesting;
> - Social Engineering Pentesting;
> - Pentest Program Management;
> - IoT Ecosystem Testing;
> - Red Team Assessment;
> - Digital Risk Assessment;
> - Secure Code Review.

Reviewer: ```suggestion
- Network Pentesting
- Mobile App Pentesting
- Web App Pentesting
- Wireless Network Pentesting
- Social Engineering Pentesting
- Pentest Program Management
- IoT Ecosystem Testing
- Red Team Assessment
- Digital Risk Assessment
- Secure Code Review
```
